File: src/main/java/parser/TokenType.java
package parser;

public enum TokenType implements ITokenType {

    NUMBER,

    WORD,
    TEXT,
    ARROW,

    EQ,


    BAR,
    BARBAR,
    AMP,
    AMPAMP,

    LBRACE,
    RBRACE,

    LBRACKET,
    RBRACKET,

    LPAREN,
    RPAREN,

    SEMI_COLON,
    COMMA,

    EOF
}

File: src/main/java/parser/TokenType.kt
package parser

enum class TokenType : ITokenType {

    NUMBER,

    WORD,
    TEXT,
    ARROW,

    EQ,


    BAR,
    BARBAR,
    AMP,
    AMPAMP,

    LBRACE,
    RBRACE,

    LBRACKET,
    RBRACKET,

    LPAREN,
    RPAREN,

    SEMI_COLON,
    COMMA,

    EOF
}
